lib/src/ruby/posix/api.cc

```diff
@@ -1,13 +1,12 @@
 #include <internal/ruby/api.hpp>
 
 using namespace std;
-using namespace facter::util;
```

Comment:
Can probably add a `using namespace leatherman::dynamic_library` here instead of qualifying it.